.cirrus.yml

```diff
@@ -0,0 +1,7 @@
+container:
+  image: node:latest
+
+check_code_committed_task:
+  check_script:
+    - node scripts/check_code_committed.js "npm run build"
```

Comment:
Does the `npm run build` needs to be invoked from within the `node`? What about invoking the npm directly from YML. And then call the action without any parameters (and without juggling with IO and arguments) just to do the git magic.